editoast/src/views/projects.rs

```diff
@@ -394,6 +400,33 @@ pub mod tests {
         assert_eq!(project.name, project_name);
     }
 
+    #[rstest]
+    async fn project_post_should_fail_when_authorization_is_enabled() {
+        let pool = DbConnectionPoolV2::for_tests();
+        let app = TestAppBuilder::new()
+            .db_pool(pool)
+            .core_client(CoreClient::Mocked(MockingClient::default()))
+            .enable_authorization(true)
```

Comment:
```suggestion
            .enable_authorization(true)
            .user(&user, &[BuiltinRole::OpsRead])
```

a `fn user(self: TestAppBuilder, &userInfo, impl IntoIterator<Item = BuiltinRole>)` calling `def_user` after building would be convenient.

Reply:
I made some changes to the API. Could you take a look and let me know what you think?